app/scripts/cartodb/cartodb-api-config.js

```diff
@@ -27,6 +27,11 @@
                 avgEnergyStar: 59,
                 ghgBuildings: 60,
                 numBuildings: 1879
+            },
+            2015: {
```

Comment:
I think the intent is to address as part of #213, but it would be nice to move this config + the `module.years` in `YearsService` to a separate config so that MOS is able to make the changes necessary to update the years displayed on their own without having to dig too far into the source. Just making a note so its not forgotten.